pkg/controller/webhooks/nextcloud.go

```diff
@@ -41,3 +47,36 @@ func SetupNextcloudWebhookHandlerWithManager(mgr ctrl.Manager, withQuota bool) e
 		}).
 		Complete()
 }
+
+func (n *NextcloudWebhookHandler) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
```

Comment:
We should also validate on update, not only on create.

Reply:
Good point. Added that part as well